README.md

```diff
@@ -76,6 +76,163 @@ For more info about the API, see the [API reference](https://kontent.ai/learn/re
+      // To see how matchesTaxonomy can work see src/Utilities/CheckboxFilter
+
+      const toggleFilter = (filterName: string, filterValue: string): void => {
+          setFilter((filter) => ({
+            ...filter,
+            [filterName]: filter[filterName].includes(filterValue)
+            ? filter[filterName].filter((x: string) => x !== filterValue)
+            : [...filter[filterName], filterValue],
+            }));
+      };.
+
+      return (
+          <div>
+              ...
+              <CheckboxFilter
+              ...
+              onChange: (event) => toggleFilter('filterVariable1', event.target.id),
+              />
+           ...
+              <ItemListing
+               items={ items[language].filter((item: ItemDTO) =>matches(coffee)) }
+               />
+           ...
+           </div>
+      );
+}
+```
+
+### Localization
+
+In Kontent each language is identified by codename, in case of this project it is `en-US` and `es-ES`.
+
+#### Resoure strings
+
+Not every text of the application must be stored in Kontent.ai. Some strings, such as button texts, navigation texts and so on, can be stored locally. For those texts we use [React-intl](https://formatjs.io/docs/getting-started/installation/). For every language we have created JSON file in `src/Localization` folder. As we use `React-intl` it can not parse nested JSON objects and therefore the format of files is `key:value`. To load all files from `src/Localization` folder we have prepared a script, see `src/utilities/LocalizationLoader.ts`.
+
+```json
+// en-US.json
+{
+  "LatestArticles.title": "Latest articles",
+  "LatestArticles.noTitleValue": "(Article has no title)",
+  "LatestArticles.noTeaserValue": "(Article has no teaser image)",
+  "LatestArticles.noSummaryValue": "No summary filled"
+  // ...
+}
+```
+
+#### Prefixes and Localizable Url slugs
+
+The language prefix is obtained from URL in the `LocalizedApp.tsx` and then it is propagated via IntlProvider to the whole application. Content language is then adjusted by modifying `Client` with `languageParameter()` method to obtain items in specific language. By default it uses [language fallbacks](https://kontent.ai/learn/tutorials/manage-kontent/projects/set-up-languages/#a-language-fallbacks) set up in the project.
+
+```typescript
+const Component: React.FC = () => {
+  const { locale: language } = useIntl();
+
+  useEffect(() => {
+    const query = Client.items<ItemDTO>()
+      .type(projectModel.contentTypes.itemDTO.codename);
+
+    if (language) {
+      query.languageParameter(language);
+    }
+    ...
+```
+
+You might want to request items based on the url slugs. For more information how it works in Kontent see this [link](https://kontent.ai/learn/tutorials/develop-apps/get-content/localized-content-items/#a-get-items-by-localized-url-slug). We have provided an example in this application for `src/Pages/About.tsx' page.
+

```

Comment:
```suggestion

> The showcase is not ideal, because it is using a combination of the language prefix and localizable solution is not ideal. You should try to stick with one of the approaches. Because it is hard to define the behavior for language setting clash i.e. `/<EN-PREFIX>/articles/<ES-URL-SLUG>`.
```